### PR TITLE
Fix bugs

### LINKED ISSUE #17: Change to WesleySurfaceResistance
https://github.com/EarthSciML/AtmosphericDeposition.jl/blob/0fae86b85024981b930c473aa59434c1805f1ac3/src/wesley1989.jl#L174

Because we might have other surface resistance algorithms later.

### LINKED ISSUE #18: Delete commented out code
https://github.com/EarthSciML/AtmosphericDeposition.jl/blob/0fae86b85024981b930c473aa59434c1805f1ac3/src/wesley1989.jl#L93

(It will just confuse people)

### PR DESCRIPTION
**Pull request by jialinl6**
Fixes #18 
Fixes #17 
Fixes https://github.com/EarthSciML/AtmosphericDeposition.jl/issues/21

### CODE REVIEW

File: docs/src/dry_deposition.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+## Model
+This is an implementation of a box model used to calculate changes in gas species concentration due to dry deposition.
+
+## Running the model
+Here's an example of how concentration of different species, such as SO<sub>2</sub>, O<sub>3</sub>, NO<sub>2</sub>, NO, H<sub>2</sub>O<sub>2</sub> and CH<sub>2</sub>O change due to dry deposition. 
+
+We can create an instance of the model in the following manner:
+```@example 1
+using AtmosphericDeposition
+using ModelingToolkit
```
Comment (ctessum) [inline docs/src/dry_deposition.md:11]: Need to add ModelingToolkit etc to https://github.com/EarthSciML/AtmosphericDeposition.jl/blob/main/docs/Project.toml

Comment (jialinl6) [reply docs/src/dry_deposition.md:11]: Thank you for pointing that out! I will remember to update the "Project.toml" file in the "docs" file.

**Comment by jialinl6** [comment]
Fixes #21

**Comment by ctessum** [comment]
> Fixes #21

This needs to go in the initial description of the pull request, otherwise it doesn't work. The general workflow is to create one pull request for each issue.

**Pull request closed without merging**